### PR TITLE
Automatically import Twine for Bindings

### PR DESCRIPTION
**Pull request by bouk**
@lemonmade for after https://github.com/Shopify/shopify/pull/76142

### CODE REVIEW

**Comment by lemonmade** [comment]
:+1:

File: packages/esify/lib/config.js [hunk 1]
```diff
@@ -35,9 +35,12 @@ module.exports = function loadConfig() {
         NProgress: 'NProgress',
         FastClick: 'shopify-fastclick',
         Clipboard: 'clipboard',
+        Twine: 'twine',
+        Bindings: 'twine',
       },
       renameIdentifiers: {
         jQuery: '$',
+        Bindings: 'Twine',
```
Comment (bouk) [inline packages/esify/lib/config.js]: @lemonmade also added this

Comment (lemonmade) [inline packages/esify/lib/config.js]: @Shopify/tnt are we cool with moving back to `Twine` as the main name for accessing Twine (over `Bindings`)?

Comment (tsov) [inline packages/esify/lib/config.js]: Yes, I think we should 🔥 `Bindings`

Comment (lemonmade) [inline packages/esify/lib/config.js]: 👍

**Pull request merged**